dev/TabView/TabView.xaml

```diff
@@ -704,6 +677,44 @@
                                 Style="{ThemeResource TabViewCloseButtonStyle}"
                                 HighContrastAdjustment="None" />
                         </Grid>
+
+                        <Path x:Name="LeftRadiusRenderArc"
+                            x:Load="False"
+                            Fill="{ThemeResource CardStrokeColorDefault}"
+                            VerticalAlignment="Bottom"
+                            Visibility="Collapsed"
+                            Margin="-4,0,0,0"
+                            Height="4"
+                            Width="4"
+                            Data="M4 0C4 1.19469 3.47624 2.26706 2.64582 3H0C1.65685 3 3 1.65685 3 0H4Z" />
+
+                        <Path  x:Name="LeftRadiusRenderTriangle"
+                            x:Load="False"
+                            Fill="{ThemeResource TabViewItemHeaderBackgroundSelected}"
+                            VerticalAlignment="Bottom"
+                            Margin="-4,0,0,0"
+                            Visibility="Collapsed"
+                            Data="M0 4H4V0C4 2.20914 2.20914 4 0 4Z" />
+
+                        <Path x:Name="RightRadiusRenderArc"
+                            x:Load="False"
+                            Grid.Column="2"
+                            Visibility="Collapsed"
+                            Fill="{ThemeResource CardStrokeColorDefault}"
+                            VerticalAlignment="Bottom"
+                            Margin="0,0,-4,0"
+                            Height="4"
+                            Width="4"
+                            Data="M0 0C0 1.19469 0.523755 2.26706 1.35418 3H4C2.34315 3 1 1.65685 1 0H0Z"/>
+
+                        <Path x:Name="RightRadiusRenderTriangle"
+                            x:Load="False"
+                            Grid.Column="2"
+                            Visibility="Collapsed"
+                            Fill="{ThemeResource TabViewItemHeaderBackgroundSelected}"
+                            VerticalAlignment="Bottom"
+                            Margin="0,0,-4,0"
+                            Data="M4 4H0V0C0 2.20914 1.79086 4 4 4Z"/>
```

Comment:
Why did you move those lines?

Reply:
@chingucoding to give the arcs bigger zIndex so they can draw on top of the tab container :)

Reply:
Did it have to draw on top of the container? I thought we moved it out wide enough.

Reply:
@chingucoding with this PR's proposal - we want to remove the gradient on the stroke color, meaning that the Tab border will go all the way to the bottom instead of nicely connecting at 4px height with the arcs on the side. In order to mitigate this, we can draw arks on top of the container so the arks themselves will cover the bottom part of the border on the sides so it's not visible:)

Reply:
I see, that makes sense.